javascript-localization.pug

```diff
@@ -302,24 +303,36 @@ block content
             p Decide which JavaScript framework to use. Then install the i18next npm-package and next add the "i18next.init" code to your application. Follow the <a href="#start" title="javascript example">JavaScript example</a> to take your first steps with i18next.
 
 
-  //- section.section-white.conclusion#conclusion(style="padding:0")
-  //-   .container
-  //-     .row
-  //-       .col-lg-12
-  //-         .section-heading
-  //-       h2 Conclusion
-  //-       p We hope you found everything out of this article, just need to switch language for an application or complete translation-management-solution for bigger projects. Plan and classify the preconditions well to have a good result for your overall project in terms of time and money. <b>Best for JavaScript localization</b> is to start with the <a href="#bestpractices" title="best practices questions">best practices questions</a> we provided above, if you decided you need a one-time-solution then you follow the guides and video instructions for selected framework. If you have more questions or want to use a more-qualified way, <a href="https://www.locize.app/register" title="Register for free">register for free</a> to locize or contact our support team at <a href="mailto:support@locize.com">support@locize.com</a> for any questions you may have. <b>In any case the library to choose is "i18next"</b> because of its richness (splitting translations to multiple files, string interpolation, plugins for <b>detect language</b> for most environments and much more), maturity (for any case) and extensibility (for all modern JavaScript framework, jquery, laravel and more). For example if you are a game developer, our <a href="/?lng=en" title="localization management platform">localization management platform</a> is also for you.
+  section.section-white.conclusion#conclusion(style="padding:0")
+   .container
+     .row
+       .col-lg-12
+         .section-heading
+          h2 Now it's your turn
+          p We hope you found everything out of this step-by-step-guide. <b>Best for javascript localization</b> is to start with the <a href="#bestpractices" title="best practices questions">best practices questions</a> we provided above. There are several methods to choose from in order to successfully localize your project with JavaScript: First of all, if you only need to switch the language for an application, it might be easier to use a javascript translation library like i18next or locizify script from us. If you have a huge project with various components to be internationalized and you want to keep up with time, then it might be a good opportunity to use a complete translation management solution like Locize. 
```

Comment:
We hope **this step-by-step-guide was helpful**...

...like i18next or **our** locizify script...

_if locize is used inside of a sentense (not at the beginning, write it lowercase)_